Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+#!groovy
+
+node {
+  try {
+    // Checkout the proper revision into the workspace.
+    stage('checkout') {
+      checkout scm
+    }
+
+    env.AWS_DEFAULT_REGION = 'us-east-1'
+    env.RF_DOCS_BUCKET = 'rasterfoundry-staging-docs-site-us-east-1'
+
+    // Use the production docs bucket for releases.
+    if (env.BRANCH_NAME.startsWith('release/')) {
+      env.RF_DOCS_BUCKET = 'rasterfoundry-production-docs-site-us-east-1'
+    }
+    // Execute `cibuild` wrapped within a plugin that translates
+    // ANSI color codes to something that renders inside the Jenkins
+    // console.
+    stage('cibuild') {
+      wrap([$class: 'AnsiColorBuildWrapper']) {
+        sh 'scripts/cibuild'
+      }
+    }
+
+    // if (env.BRANCH_NAME == 'develop' || env.BRANCH_NAME.startsWith('release/')) {
```

Comment:
Reminder to update here, [L35](https://github.com/azavea/raster-foundry-docs/pull/5/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R35), and [L44](https://github.com/azavea/raster-foundry-docs/pull/5/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R44)